src/app/language-support/page.tsx

```diff
@@ -2,24 +2,43 @@
 
 import { getAllInterpretation } from '@/api/supabase/queries/interpretation';
 import { getAllDocuments } from '@/api/supabase/queries/documentTranslation';
+import { getAllCases } from '@/api/supabase/queries/cases';
 import { useEffect, useState } from 'react';
-import { DocumentTranslation, Interpretation } from '@/types/schema';
+import {
+  DocumentTranslation,
+  Interpretation,
+  CaseListing,
+} from '@/types/schema';
 
 export default function Page() {
-  const [docListings, setDocListings] = useState<DocumentTranslation[]>([]);
-  const [intListings, setIntListings] = useState<Interpretation[]>([]);
+  type AllLanguageSupport = DocumentTranslation | Interpretation | CaseListing;
```

Comment:
I think this would help for other parts of the app as well, can we put this in the schema?